src/main/kotlin/no/nav/amt/arena/acl/repositories/ArenaGjennomforingRepository.kt

```diff
@@ -0,0 +1,96 @@
+package no.nav.amt.arena.acl.repositories
+
+import no.nav.amt.arena.acl.domain.kafka.amt.AmtGjennomforing
+import no.nav.amt.arena.acl.domain.kafka.amt.AmtTiltak
+import no.nav.amt.arena.acl.utils.*
+import no.nav.amt.arena.acl.utils.DatabaseUtils.sqlParameters
+import org.springframework.jdbc.core.RowMapper
+import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate
+import org.springframework.stereotype.Component
+import java.util.*
+
+@Component
+class ArenaGjennomforingRepository (
+	val template: NamedParameterJdbcTemplate
+) {
+	val rowMapper = RowMapper { rs, _ ->
+		AmtGjennomforing(
```

Comment:
Jeg tror kanskje jeg ville laget en egen DBO for å representere gjennomføringen i databasen